DEBIAN/control

```diff
@@ -0,0 +1,17 @@
+Package: plank
+Version: 0.11.100
+Priority: optional
+Section: utils
+Maintainer: zquestz (https://github.com/zquestz)
+Architecture: amd64
+Depends: bamfdaemon, libbamf3-2t64, libgee-0.8-2, libgnome-menu-3-0
+Homepage: https://github.com/zquestz/plank-reloaded
+Tag: uitoolkit::gtk
+APT-Manual-Installed: yes
+Description: Elegant, simple, clean dock
```

Comment:
Description should be for Plank Reloaded as well. You can grab copy from the README.